charts/flagger/values.yaml

```
@@ -195,8 +195,12 @@ podDisruptionBudget:
   enabled: false
   minAvailable: 1
 
+# Additional labels to be added to pods
 podLabels: {}
 
+# Additional labels to be added to deployments
+deploymentLabels: { }
```

```suggestion
deploymentLabels: {}
```

this typo missed my eye.